bundles/io/org.eclipse.smarthome.io.transport.mqtt/src/main/java/org/eclipse/smarthome/io/transport/mqtt/MqttService.java

```diff
@@ -12,84 +12,52 @@
  */
 package org.eclipse.smarthome.io.transport.mqtt;
 
-import java.util.Collections;
-import java.util.List;
 import java.util.Map;
-import java.util.concurrent.ConcurrentHashMap;
-import java.util.concurrent.CopyOnWriteArrayList;
 
-import org.apache.commons.lang.StringUtils;
 import org.eclipse.jdt.annotation.NonNullByDefault;
 import org.eclipse.jdt.annotation.Nullable;
 import org.eclipse.smarthome.io.transport.mqtt.internal.MqttBrokerConnectionServiceInstance;
-import org.osgi.framework.Constants;
-import org.osgi.service.cm.ConfigurationException;
-import org.osgi.service.component.annotations.Component;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 
 /**
  * This service allows you to enumerate system-wide configured Mqtt broker connections. You do not need this service
  * if you want to manage own/local Mqtt broker connections. If you add a broker connection, it will be
  * available immediately for all MqttService users. A removed broker connection may still be in use by consuming
  * services.
  *
- * Added/removed connections are not permanent. If the service is shutdown/restored it will
- * contain the system-wide configured connections again (usually via text files).
+ * Added/removed connections are not permanent. Use {@link MqttBrokerConnectionServiceInstance} to configure permanent
+ * connections.
  *
- * System broker connections are not configured via this service. See {@link MqttBrokerConnectionServiceInstance}
- * instead.
- *
- * @author David Graeff - Added/Removed observer interface, Add/Remove/Enumerate broker connections.
- * @author Davy Vanherbergen
- * @author Markus Rathgeb - Synchronize access to broker connections
+ * @author David Graeff - Initial contribution
  */
-@Component(immediate = true, service = {
-        MqttService.class }, configurationPid = "org.eclipse.smarthome.mqtt", property = {
-                Constants.SERVICE_PID + "=org.eclipse.smarthome.mqtt" })
 @NonNullByDefault
-public class MqttService {
-    private final Logger logger = LoggerFactory.getLogger(MqttService.class);
-    private final Map<String, MqttBrokerConnection> brokerConnections = new ConcurrentHashMap<String, MqttBrokerConnection>();
-    private final List<MqttServiceObserver> brokersObservers = new CopyOnWriteArrayList<>();
+public interface MqttService {
```

Comment:
Remove the public keyword from all methods as methods are already public by definition.